CHANGELOG.md

```diff
@@ -1,3 +1,147 @@
+<a name="2.0.0-beta.2"></a>
+# [2.0.0-beta.2 flannel-papaya](https://github.com/angular/material2/compare/2.0.0-beta.1...2.0.0-beta.2) (2017-02-15)
+
+### Breaking changes from beta.1
+* Styling is no longer prefixed by `md-`. All styling is now prefixed by `mat-` so that apps can upgrade from AngularJS Material to Angular Material without styling conflicts between the two library components.
+See ([#2790](https://github.com/angular/material2/issues/2790)) for the details on the code change and some useful regular expressions that can help migrate styles.
+* Checkbox tab index input has been changed from `tabindex` to `tabIndex`. ([#2953](https://github.com/angular/material2/issues/2953))
+* Ripple no longer has the `mdRippleBackgroundColor` input to change the background color. ([#2859](https://github.com/angular/material2/issues/2859))
+* The deprecated use of `<md-input>` and `<md-textarea>` has been removed. Use `mdInput` on an input within a `md-input-container`. `md-prefix` and `md-suffix` are now `mdPrefix` and `mdSuffix`. ([#2788](https://github.com/angular/material2/issues/2788))
+* The deprecated use of `<md-sidenav-layout>` has been removed. Use `<md-sidenav-container>` instead. ([#2283](https://github.com/angular/material2/issues/2283))
+* Input floating placeholder input has changed from a boolean (`true` and `false`) to a state (`always`, `never`, and `auto`). For details on when to use which state, see  ([#2585](https://github.com/angular/material2/issues/2585))
```

Comment:
Also important to note that it changed from `floatingPlaceholder` to `floatPlaceholder`.